backend/tests/Squidex.Domain.Apps.Entities.Tests/Contents/DomainObject/Guards/GuardContentTests.cs

```diff
@@ -247,6 +295,34 @@ public async Task CanChangeStatus_should_not_throw_exception_if_status_flow_vali
             await GuardContent.CanChangeStatus(command, content, contentWorkflow, contentRepository, schema);
         }
 
+        [Fact]
+        public async Task CanChangeOwnStatus_should_not_throw_exception_if_has_permission()
+        {
+            var schema = CreateSchema(false);
+            var refToken = new RefToken("string", "string");
+            var content = CreateOwnContent(status: Status.Draft, refToken: refToken, schema: schema);
+            var command = ChangeContentSatusCommand(content);
+
+            A.CallTo(() => contentWorkflow.CanMoveToAsync(content, content.Status, command.Status, userWithStatusChangePermission))
+                .Returns(true);
+
+            await GuardContent.CanChangeStatus(command, content, contentWorkflow, contentRepository, schema);
+        }
+
+        [Fact]
+        public async Task CanChangeOwnStatus_should_throw_exception_if_has_no_permission()
+        {
+            var schema = CreateSchema(false);
```

Comment:
Avoid code duplication in tests.

Reply:
I can refToken to move to  test class as property and use anywhere like this:
`private readonly RefToken refToken = new RefToken("string", "string");`

Do you commands, contents too?

Reply:
This does not make sense, I mean you could just create a helper method or so, but creating one field that is only needed for one test just moves the duplication from the method to the class level.